sql/auto_install.sql

```diff
@@ -1,19 +1,43 @@
+-- +--------------------------------------------------------------------+
+-- | Copyright CiviCRM LLC. All rights reserved.                        |
```

Comment:
Looks like you missed reverting changes to this file. The spacing and tabs are all messed up.